src/enums/businessTypes.ts

```diff
@@ -1,3 +1,10 @@
+/** Association types - only used for COOP for now. */
+export enum AssociationTypes {
+  COMMUNITY_SERVICE_COOPERATIVE = 'CSC',
+  HOUSING_COOPERATIVE = 'HC',
+  ORDINARY_COOPERATIVE = 'CP',
+}
```

Comment:
I'd prefer a new file for this, to keep it separate (and to make it easier to find based on filename).